### PR TITLE
Flag experimental and beta modules in the docs

### PR DESCRIPTION
**Pull request by dedemorton**
@monicasarbu I've updated docs to reflect the status of the modules shown here.

https://github.com/elastic/beats/issues/4579

A couple of questions: 
* should the ES module be flagged as experimental? beta?
* the list doesn't mention the aerospike or graphite modules. How should those be flagged?

### CODE REVIEW

File: metricbeat/module/ceph/cluster_status/_meta/docs.asciidoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
-=== Ceph cluster_health metricset
+=== Ceph cluster_status metricset
```
Comment (ruflin) [inline metricbeat/module/ceph/cluster_status/_meta/docs.asciidoc]: Good catch

**Comment by ruflin** [comment]
Thanks for adding docs for this. 

* Aerospike: Experimental
* Graphite: Experimental

For ES we don't have a beta or experimental flag, so I think it's fine as is.

**Comment by dedemorton** [comment]
@ruflin Oh wait, I see that someone added the experimental flag here, but the syntax is wrong, so it's not resolving correctly: https://www.elastic.co/guide/en/beats/metricbeat/6.0/exported-fields-elasticsearch.html. Should I mark the module as experimental?

Also, I found an option under the mysql module status metricset marked as experimental. Should I keep or remove the flag?

https://www.elastic.co/guide/en/beats/metricbeat/6.0/metricbeat-metricset-mysql-status.html#_raw_config_option

I'm done combing the docs for stuff, so that should be all. :-)

**Comment by ruflin** [comment]
@dedemorton Yes, please format it correctly. And I should add a log line about being experimental to the metricsets, because it's missing. For 6.1 we should go again through the list to see what we make beta and what we release.

For the mysql status metricsets, it's only the raw option which is experimental. This should stay in.

**Comment by dedemorton** [comment]
@ruflin This is ready for a final review.

File: metricbeat/module/dropwizard/_meta/docs.asciidoc [hunk 2]
```diff
@@ -1,4 +1,6 @@
 == Dropwizard module
 
+beta[]
```
Comment (ruflin) [inline metricbeat/module/dropwizard/_meta/docs.asciidoc]: This should also be experimental I think.

**Comment by ruflin** [comment]
@dedemorton I'm good with merging as is. I think it's good that we now have it "properly" in the docs. I think we should do a second round later on to decide which ones we make beta or release.

**Pull request merged**